packages/expect/src/jest-expect.ts

```diff
@@ -699,6 +699,42 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
       )
     },
   )
+  def(
+    ['toHaveBeenCalledBefore'],
+    function (resultSpy: MockInstance) {
+      const expectSpy = getSpy(this)
```

Comment:
We also need to validate the `resultSpy` to be a mock function

Reply:
Indeed, I will update that part.